### PR TITLE
Update it.json

### PR DESCRIPTION
**Pull request by hacklover**
Updated translations

### CODE REVIEW

File: langs/it.json [hunk 1]
```diff
@@ -47,6 +42,40 @@
     "Integrity check of resource failed": "Controllo dell'integrità delle risorse fallito",
     "Change your nickname": "Cambia il tuo nickname",
     "Connecting": "Connessione in corso",
-    "Abort": "Annulla"
+    "Abort": "Annulla",
+    "Home": "Home",
```
Comment (Lhoerion) [inline langs/it.json:46]: Shouldn't be `Home` translated too?

Comment (hacklover) [reply langs/it.json:46]: Home is well recognized as "main page" in Italian, it would be a stretch to translate it

Comment (Lhoerion) [reply langs/it.json:46]: Gotch ya, just asking (I don't know Italian)

**Review by Lhoerion — APPROVED** [review]
Thanks for your contribution

**Pull request merged**